src/site/_filters/find-tags.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const blogTags = require("../_data/blogTags");
+
+/**
+ * @param {Array<string>} tags Array of tags.
+ * @return {Array<object>} An array of tags urls, titles, and any other information.
+ */
+module.exports = (tags) => {
+  const validTags = [];
+
+  for (const tag of tags) {
+    const lowerCaseTag = tag.toLowerCase();
```

Comment:
I know you say `toLowerCase()` everywhere, but it feels odd. Can we remove this and have a better error/warning if you get a tag wrong?